src/css/custom.css

```diff
@@ -370,12 +399,16 @@ span.callout + p::after {
   width: 100%;
 }
 
+/* Add space and thin border to images, remove drop shadow */
 .theme-doc-markdown img {
   border-style: solid;
-  border-color: lightgray;
+  border-color: #dedede;
   border-width: 1px;
-  box-shadow: 4px 4px 4px 1px lightgray;
+  box-shadow: none;
```

Comment:
```suggestion
```

non-blocking (and untested): 

I'm pretty sure you can just remove this property, if we are getting rid of the shadow effect.

Reply:
Yep, done! 👍